core/local/steps/win_detect_move.js

```diff
@@ -30,23 +30,18 @@ module.exports = {
 // TODO add unit tests and logs
 // TODO check that a file/dir created and removed just after is not seen as a move
 
+function loop (buffer /*: Buffer */, opts /*: { pouch: Pouch } */) /*: Buffer */ {
+  const out = new Buffer()
+  winDetectMove(buffer, out, opts.pouch)
+    .catch(err => log.error({err}))
+  return out
+}
```

Comment:
If this commit only include refactoring, it can be reviewed/merged later in a separate PR.